diagnostic.md

```diff
@@ -14,7 +17,7 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+return $("li").get();
```

Comment:
Not quite. Just `$('li');` would work. Also, TWO ways. Another would be `$('ol > li')`